recipe/meta.yaml

```diff
@@ -10,19 +10,22 @@ source:
   sha256: 0dbd14a8aef6636764f88a8fd1fcc9a91921e5c50356e6aab251782f264ae960
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
   number: 0
-
+  skip: true  # [py<38]
+  
 requirements:
   host:
     - pip
-    - python >=3.7
-    - hatchling >=1.0
+    - python
+    - hatchling >=1.5
+    - wheel
   run:
-    - python >=3.7
-    - hatchling >=1.0
+    - python
+    - hatchling >=1.5
+
 
+# Note windows tests suppressed because of problems with utils.run(shlex.split(cmd))
```

Comment:
For readability, could you please move this comment to just above the line:
`python -m pytest -k "not (test_run or test_normalize_cmd)" -vv  # [win]`